src/main/java/uk/gov/ons/census/fwmt/jobservice/transition/EmptyCreateUnitUpdate.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.ons.census.fwmt.jobservice.transition;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.common.rm.dto.ActionInstructionType;
+import uk.gov.ons.census.fwmt.common.rm.dto.FwmtActionInstruction;
+import uk.gov.ons.census.fwmt.events.component.GatewayEventManager;
+import uk.gov.ons.census.fwmt.jobservice.data.GatewayCache;
+import uk.gov.ons.census.fwmt.jobservice.service.MessageCacheService;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.InboundProcessor;
+import uk.gov.ons.census.fwmt.jobservice.service.processor.ProcessorKey;
+
+import static uk.gov.ons.census.fwmt.jobservice.config.GatewayEventsConfig.COMET_UPDATE_PRE_SENDING;
+
+@Qualifier("Update")
+@Service
+public class EmptyCreateUnitUpdate implements InboundProcessor<FwmtActionInstruction> {
+
+
+  @Autowired
+  private GatewayEventManager eventManager;
+
+  @Autowired
+  private MessageCacheService messageCacheService;
+
+  private static ProcessorKey key = ProcessorKey.builder()
+      .actionInstruction(ActionInstructionType.UPDATE.toString())
+      .surveyName("CENSUS")
+      .addressType("CE")
+      .addressLevel("U")
+      .build();
+
+  @Override
+  public ProcessorKey getKey() {
+    return key;
+  }
+
+  @Override
+  public boolean isValid(FwmtActionInstruction rmRequest, GatewayCache cache) {
+    try {
+      return rmRequest.getActionInstruction() == ActionInstructionType.UPDATE
```

Comment:
will error with CeUpdateUnitPreCreateProcessor as isValid is identical

whats this for

Reply:
See comment above.